hugo/content/about.md

```diff
@@ -0,0 +1,66 @@
+---
+title: About
+toc: true
+---
+
+# About the project
+
+Mumble is a [free, open source](https://en.wikipedia.org/wiki/Free_and_open-source_software), low latency, high quality voice chat application.
+
+## Source Code & Development
+
+The Source Code and further information can be found in our [GitHub repo](https://github.com/mumble-voip/mumble).
+
+For other Mumble Projects, see [Overview of Repositories on GitHub](https://github.com/mumble-voip/).
+
+## Licenses
+
+Mumble is [Free and Open-Source Software](https://en.wikipedia.org/wiki/Free_and_open-source_software) and licensed under a BSD-style [License](https://www.mumble.info/LICENSE).
+
+All Licenses are also listed in the GitHub repositories, see: [Main Mumble License](https://github.com/mumble-voip/mumble/blob/master/LICENSE) & [Overview of other Mumble Projects](https://github.com/mumble-voip/)
+
+
+# About us
+
+We are a team of voluntary developers - both professional and non-professional - from various countries, including Italy, Germany, USA and more.
+
+## Core Team
+
+Active Team Members are:
+
+| Name: | Github Profile: | Assignment: |
+| --- | --- | --- |
+| Davide Beatrici | [davidebeatrici](https://github.com/davidebeatrici) | Core developer |
+| Robert Adam | [Krzmbrzl](https://github.com/Krzmbrzl) | Core developer, GitHub Management |
+| Nik Reist | [ZeroAbility](https://github.com/ZeroAbility) | Release Management |
+| Jan Klass | [Kissaki](https://github.com/Kissaki) | Community work |
+| Natenom | [Natenom](https://github.com/Natenom) | Testing |
```

Comment:
Example table.

Reply:
In case of Natenom I would leave the real name entry blank.

Reply:
:thinking: I think leaving an empty space is not so good...
How about clarifying that it is a nickname?
Like `Natenom (Nickname)`

Reply:
If you don't want to leave it blank, fill the table cell with a single `-` to symbolize that the real name is not provided.

Reply:
Well, if you want it like that.